lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -398,7 +398,7 @@ def _rows_for_users(self, context, users):
         Returns a list of rows for the given users for this report.
         """
         with modulestore().bulk_operations(context.course_id):
-            bulk_context = _CourseGradeBulkContext(context, users)
+            # bulk_context = _CourseGradeBulkContext(context, users)
```

Comment:
Let's keep this line here since we still want to call `PersistentCourseGrade.prefetch`.